lib/git-lfs-s3.rb

```diff
@@ -1,6 +1,6 @@
-require 'sinatra/base'
-require 'aws-sdk'
-require 'multi_json'
+require "sinatra/base"
+require "aws-sdk"
+require "multi_json"
```

Comment:
Because there were single and double quotes with two different require blocks. I just used one style.